eigen.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
you don't need a manifest for a signle module like that. It's harder to maintain / read